domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -168,6 +168,10 @@ public LocalDate getServiceDate() {
         return TotalAmountCalculator.calculateInterestForClaim(this);
     }
 
+    public Optional<BigDecimal> getTotalInterestTillDateOfIssue() {
```

Comment:
You will need to add in `@JsonIgnoreProperties` to ignore them when deserialize

Reply:
👍

Reply:
Always forget about jackson until something breaks xd